pinot-controller/src/main/java/org/apache/pinot/controller/recommender/io/InputManager.java

```diff
@@ -71,7 +80,8 @@
   // Basic input fields
 
   public Long _qps = DEFAULT_QPS;
-  public Long _numMessagesPerSecInKafkaTopic = DEFAULT_NUM_MESSAGES_PER_SEC_IN_KAFKA_TOPIC; // messages per sec for kafka to consume
+  public Long _numMessagesPerSecInKafkaTopic = DEFAULT_NUM_MESSAGES_PER_SEC_IN_KAFKA_TOPIC;
+      // messages per sec for kafka to consume
```

Comment:
Indentation is off, or is the github UI showing it as off?

Reply:
Hmm.. Auto-format.. Let me revert this since it is irrelevant